pyproject_metadata/__init__.py

```diff
@@ -386,7 +404,7 @@ class StandardMetadata:
     def __post_init__(self) -> None:
         self.validate()
 
-    def validate(self) -> None:
+    def validate(self, *, show_warnings: bool = True) -> None:
```

Comment:
Technically, the warnings are emitted, not show, thus the argument name is not precise. I would call it simply `warn`, but it doesn't matter.

Reply:
Went with `warn`.